tests/nl/homeassistant_HassGetState.yaml

```diff
@@ -1,2 +1,55 @@
 language: nl
-tests: []
+tests:
+  - sentences:
+      - "Wat is de buitentemperatuur"
+    intent:
+      name: HassGetState
+      slots:
+        name: "Buitentemperatuur"
+    response: "Buitentemperatuur is 15 °C"
+
+  - sentences:
+      - "Staat de slaapkamerlamp aan?"
+    intent:
+      name: HassGetState
+      slots:
+        name: "Slaapkamerlamp"
+        state: "on"
+    response: "Nee, slaapkamerlamp is off"
```

Comment:
Dit moet "Nee, slaapkamerlamp is uit" zijn. Ik denk dat we iig `off` kunnen afvangen ?

Reply:
Ik kan `off` en `on` inderdaad wel vertalen in het template.